wd_core.au3

```diff
@@ -854,22 +855,24 @@ Func _WD_ExecuteScript($sSession, $sScript, $sArguments = Default, $bAsync = Def
 
 		$sResponse = __WD_Post($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/execute/" & $sCmd, $sData)
 		Local $iErr = @error
```

Comment:
I just notice that we can get here different error results 
so IMO `; Return values .:` in header should be suplemented.

or I'm wrong ?

Reply:
I believe this is already covered in the current header, which states --

> ;                  Failure - Response from web driver in JSON format and sets @error to value returned from __WD_Post()
;                            If script is executed successfully but $vSubNode isn't found, then "" (empty string) and sets @error to $_WD_ERROR_RetValue
;                            If $vSubNode isn't valid, then "" (empty string) and sets @error to _WD_ERROR_InvalidArgue

I think `sets @error to value returned from __WD_Post()` could be considered ambiguous because @error isn't actually set to the returned value; it's set to the error code returned by __WD_Post.

Reply:
> I think `sets @error to value returned from __WD_Post()` could be considered ambiguous because @error isn't actually set to the returned value; it's set to the error code returned by __WD_Post

Please make more proper description

Reply:
What do you think changing this --

> ;                  Failure - Response from web driver in JSON format and sets @error to value returned from __WD_Post()

to this --

> ;                  Failure - Response from web driver in JSON format and maintains @error value originally set by __WD_Post()

??

Reply:
Ok